pkg/services/ngalert/remote/remote_primary_forked_alertmanager.go

```diff
@@ -41,6 +41,13 @@ func (fam *RemotePrimaryForkedAlertmanager) ApplyConfig(ctx context.Context, con
 
 // TODO: save the new configuration hash in memory.
 func (fam *RemotePrimaryForkedAlertmanager) SaveAndApplyConfig(ctx context.Context, config *apimodels.PostableUserConfig) error {
+	if err := fam.remote.SaveAndApplyConfig(ctx, config); err != nil {
+		return err
+	}
+
+	if err := fam.internal.SaveAndApplyConfig(ctx, config); err != nil {
+		fam.log.Error("Error applying config to the internal Alertmanager", "err", err)
+	}
```

Comment:
Do we mean to ignore the error and only log it? It's fine if that's the intention, I would add a comment here as to why we're doing this as it's a bit unusual.